app/controllers/edit.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    edit(_id) {
+      const data = this.model;
+      fetch(`http://tiny-tn.herokuapp.com/collections/runs-hs/${_id}`, {
+          headers: {
+            'Content-type': 'application/json',
+            'Accept': 'application/json'
+          },
+          method: 'PUT',
+          body: JSON.stringify(data)
+        });
+        this.transitionToRoute("detail");
```

Comment:
Transition to route detail needs a id to go to since the detail requires an `id` similar to how the `link-to` works